rubocop-rspec_rails.gemspec

```diff
@@ -35,4 +35,5 @@ Gem::Specification.new do |spec|
   }
 
   spec.add_runtime_dependency 'rubocop', '~> 1.40'
+  spec.add_runtime_dependency 'rubocop-rspec', '~> 3.0.0.pre'
```

Comment:
@ydah Is it ok to have this restriction in the gemspec, and then we do a prerelease of rubocop-rspec_rails? Or should we just wait for rubocop-rspec v3.0.0 proper?

Reply:
I think pre-pre should be fine